playwright-tests/tests/landing-page.spec.js

```diff
@@ -0,0 +1,169 @@
+import { expect, test } from "@playwright/test";
+import { ROOT_SRC } from "../util/constants";
+
+test.describe("Navbar and it's elements are visible and redirected to their respective pages", () => {
+  test.beforeEach(async ({ page }) => {
+    await page.goto(`/${ROOT_SRC}`);
+  });
+  test("Home", async ({ page }) => {
+    const homeTab = page.locator("span", { hasText: "Home" });
+    await homeTab.click();
+    expect(page.url()).toContain("?page=home");
+  });
+
+  test("Activity", async ({ page }) => {
+    const activityTab = page.locator("span", { hasText: "Activity" });
+    await activityTab.click();
+    expect(page.url()).toContain("?page=activity");
+  });
+  test("Projects", async ({ page }) => {
+    const projectsTab = page.getByRole("link", {
+      name: "Projects",
+      exact: true,
+    });
+    await projectsTab.click();
+    expect(page.url()).toContain("?page=projects");
+  });
+  test("Resources", async ({ page }) => {
+    const resourcesTab = page.locator("span", { hasText: "Resources" });
+    await resourcesTab.click();
+    expect(page.url()).toContain("?page=resources");
+  });
+});
+
+test.describe("Start project redirection for logged in users ", () => {
+  test.use({
+    storageState: "playwright-tests/storage-states/wallet-connected.json",
+  });
+  test.beforeEach(async ({ page }) => {
+    await page.goto(`/${ROOT_SRC}`);
+  });
+  test("User is logged in", async ({ page }) => {
+    const startProjectButton = page.getByRole("button", {
+      name: "Start Project",
+    });
+    await expect(startProjectButton).toBeVisible();
+    await startProjectButton.click();
+    expect(page.url()).toContain("?page=projects&tab=editor");
+  });
+});
+
+test.describe("Start project redirection for logged out users", () => {
+  test.use({
+    storageState: "playwright-tests/storage-states/wallet-not-connected.json",
+  });
+  test.beforeEach(async ({ page }) => {
+    await page.goto(`/${ROOT_SRC}`);
+  });
+  test("User is not logged in", async ({ page }) => {
+    const startProjectButton = page.getByRole("button", {
+      name: "Start Project",
+    });
+    await expect(startProjectButton).toBeVisible();
+    await startProjectButton.click();
+    expect(page.url()).toContain("?page=projects&tab=editor");
+  });
+});
+
+test.describe("Explore Projects redirection for logged in user", () => {
+  test.use({
+    storageState: "playwright-tests/storage-states/wallet-connected.json",
+  });
+  test.beforeEach(async ({ page }) => {
+    await page.goto(`/${ROOT_SRC}`);
+  });
+  test("User is logged in", async ({ page }) => {
+    const exploreProjectsButton = page.getByRole("button", {
+      name: "Explore Project",
+    });
+    await expect(exploreProjectsButton).toBeVisible();
+    await exploreProjectsButton.click();
+    expect(page.url()).toContain("?page=projects");
+  });
+});
+
+test.describe("Explore Projects for logged out user", () => {
+  test.use({
```

Comment:
let's have two tests with subtests like this: 
1) User not logged in:
     - Start Project action
2) User logged in: 
     - Start Project action
     - Explore project action
     - all other links redirection
     - Social Media links
(you can checkout Elliot's PR for reference)

Reply:
Sure will implement this structure.

Reply:
Implemented it.